main.js

```diff
@@ -1,7 +1,8 @@
 (function(){
   var node = document.querySelector("div");
   var app = Elm.Main.embed(node, {
-      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : ""
+      eventId: window.location.href.split('/events/')[1] ? window.location.href.split('/events/')[1].split('/')[0] : "",
```

Comment:
Ok. Let's take this conversation here: https://github.com/OxfordAbstracts/elm-program-builder/issues/51